server/lib/chart.js

```diff
@@ -0,0 +1,127 @@
+const _ = require('lodash');
+const util = require('./util');
+
+module.exports = {
+  barChar,
+};
+
+/**
+ * @function barChar
+ * This function help to skip some manipulation for respecting chartjs lib format...
+ */
+
+function barChar(params) {
+  const {
+    label, data, item, yAxesLabelString,
+    xAxesLabelString, canvasId,
+  } = params;
+
+  const chart = {
+    labels : [],
```

Comment:
Since we know this is always going to be for a period, can we just generate the values of 0 - 30 days for the labels?  Any days without data will be 0?

Reply:
This function will also be used for other purposes. I created a function to make it reusable.
While creating series for chart.js, all items must have a value for every label.
Imagine on march 2th, a user want to know the stock consumption. The series will have a lot value for nothing.

Let take an example

```js
labels = ['01','02'];
```

```js
items = ['inventory1', 'inventory2', 'inventory3']
```

the series (datasets) should be
```js
[{
   label : 'inventory1',
   data : [10, 3]
 },
{ 
  label : 'inventory2',
  data : [5, 45]
},
{
  label : 'inventory3',
  data : [20, 0]
}];
```

every dataset should have the an array(data) that contains the value for each label
so the array's length must be equals to labels length